catalog-info.yaml

```diff
@@ -0,0 +1,24 @@
+# Catalog entry for Backstage [backstage.io]
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: eco-goinfra
+  title: eco-goinfra
+  description: Ecosystem QE framework
+  links:
+    - title : Ginkgo
+      url: https://onsi.github.io/ginkgo/
+  tags:
+    - go
+    - ginkgo
+    - test-framework
+    - product-quality
+    - openshift
+  namespace: quality-community
+  annotations:
+    github.com/project-slug: openshift-kni/eco-goinfra
+    backstage.io/techdocs-ref: https://github.com/openshift-kni/eco-goinfra/wiki
```

Comment:
Should we have something here?

Reply:
From what I see in other examples we should point to the documentation. 

Maybe it is better to have it point to the README ?

Reply:
Yeah I think the readme makes sense

Reply:
Updated to use README.